MaxText/kernels/megablox/gmm.py

```diff
@@ -551,7 +563,7 @@ def out_transform_indices(n_i, grid_id, k_i, group_metadata, group_offset):
   flops = 2 * m * k * n
   cost_estimate = pl.CostEstimate(flops=flops, bytes_accessed=bytes_accessed, transcendentals=0)
   if lhs_quantize_dtype is not None or rhs_quantize_dtype is not None:
-    pallas_call_fn = aqt_pl.pallas_call
+    pallas_call_fn = qpl.pallas_call
```

Comment:
I heard from @KissShane that Qwix hasn't supported the tgmm quantization yet. Not sure if we'd like to hold this PR or support both till then. Otherwise, the quantization of DeepSeek will have lower perf. cc @suexu1025